### ISSUE TITLE
Upstream 0.6.0 terminfo changes to ncurses

### ISSUE THREAD

**Issue by chrisduerr** [alacritty/alacritty#4108]
The 0.6.0 version already has some terminfo changes like the colon separated form for setab/setaf.
So once the stable release is published, we need to update Alacritty's terminfo.

**Comment by kchibisov** [comment]
I'd also remind that alacritty 0.6.0 should be available in some package manager(likely suse or arch) in a moment of sending patch to ncurses, since it's tested on those distros from with package manager version.

**Comment by chrisduerr** [comment]
> I'd also remind that alacritty 0.6.0 should be available in some package manager(likely suse or arch) in a moment of sending patch to ncurses, since it's tested on those distros with version from a package manager.

I don't understand what you mean by this? Could you rephrase/clarify please?

**Comment by kchibisov** [comment]
I mean that patch will be applied to ncurses only if it could be tested, but testing will require alacritty 0.6.0, so Thomas should be able to install it somehow, and the only way they *WILL* do that is by installing alacritty from the system's package manager. Otherwise your patch won't be tested and so could be forgotten.

The last time we talked they were mentioning Arch Linux or SUSE Linux as a testing machines. And they weren't that happy about Arch one.

**Comment by kchibisov** [comment]
I'm basically trying to say, that you shouldn't send a patch right way, but you should hold it until alacritty 0.6.0 will reach major distros.

**Comment by chrisduerr** [comment]
Makes sense, especially since it will take a while (like years) until the patch is merged and distributed anyways.
